services/connectivity/messaging/src/main/java/org/eclipse/ditto/services/connectivity/messaging/validation/HostValidator.java

```diff
@@ -58,19 +58,19 @@
      */
     HostValidator(final ConnectivityConfig connectivityConfig, final LoggingAdapter loggingAdapter,
             final AddressResolver resolver) {
-        this.allowedHostnames = connectivityConfig.getConnectionConfig().getAllowedHostnames();
         this.resolver = resolver;
+        this.allowedHostnames = connectivityConfig.getConnectionConfig().getAllowedHostnames();
         final Collection<String> blockedHostnames = connectivityConfig.getConnectionConfig().getBlockedHostnames();
         this.blockedAddresses = calculateBlockedAddresses(blockedHostnames, loggingAdapter);
     }
 
     /**
      * Validate if connections to a host are allowed by checking (in this order):
      * <ul>
-     *     <li>blocklist is empty? this completely disables validation, every host is allowed</li>
-     *     <li>host is contained in allowlist? host is allowed</li>
+     *     <li>blocklist is empty this completely disables validation, every host is allowed</li>
+     *     <li>host is contained in allowlist host is allowed</li>
```

Comment:
```suggestion
     *     <li>if the blocklist is empty, this completely disables validation, every host is allowed</li>
     *     <li>if the host is contained in the allowlist, the host is allowed</li>
```

Reply:
It makes sense to extend the javadoc in this way 👍🏻